src/components/method/example_group.js

```diff
@@ -14,7 +14,11 @@ export default class ExampleGroup extends React.Component {
     return (
       <div className="examples">
         <ul className="examples-list">{
-          this.examples().map((ex) => <li>{ ex }</li>)
+          this.examples().map((ex) => {
+            let httpCode = ex.props.example.httpCode();
+
+            return <li key={httpCode + ":" + Math.random()}>{ ex }</li>;
```

Comment:
`Array.prototype.map` provides the index of the element as the second argument to the iterating function. Use that.